azure-pipelines.yml

```diff
@@ -0,0 +1,486 @@
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Windows Arm64
+    - job:
+      displayName: Windows_Arm64
+      agentOS: Windows
+      variables:
+        _AdditionalMSBuildArguments: "/p:SkipTests=true  /p:StabilizePackageVersion=$(IsStable)"
+        _PortableBuild: true
+        _TargetArchitecture: arm64
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCorePublic-Int-Pool
+          queue: Windows.10.Amd64.Open
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dotnet-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Windows Arm
+    - job:
+      displayName: Windows_Arm
+      agentOS: Windows
+      variables:
+        _AdditionalMSBuildArguments: "/p:SkipTests=true  /p:StabilizePackageVersion=$(IsStable)"
+        _PortableBuild: true
+        _TargetArchitecture: arm
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCorePublic-Int-Pool
+          queue: Windows.10.Amd64.Open
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dotnet-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+# Platform: OSX
+- template: /eng/jobs/osx-build.yml
+  parameters:
+    pool:
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        name: Hosted macOS
+      ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        name: Hosted Mac internal
+    strategy:
+      maxParallel: 5
+      matrix: 
+        OSX-x64-debug:
+          _AdditionalBuildArguments: "-PortableBuild=true -strip-symbols -SkipTests=$(SkipTests)"
+          _AdditionalMSBuildArguments: " /p:StabilizePackageVersion=$(IsStable)"
+          _BuildConfiguration: Debug
+          _PortableBuild: true
+        OSX-x64-release:
+          _AdditionalBuildArguments: "-PortableBuild=true -strip-symbols -SkipTests=$(SkipTests)"
+          _AdditionalMSBuildArguments: " /p:StabilizePackageVersion=$(IsStable)"
+          _BuildConfiguration: Release
+          _PortableBuild: true
+          _TargetArchitecture: x64
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  # Signing Validation
+  - template: /eng/jobs/validate-signing.yml
+    parameters:
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCorePublic-Int-Pool
+          queue: Windows.10.Amd64.Open
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dotnet-internal-temp
+      dependsOn: 
+          - Linux_x64_glibc
```

Comment:
It feels like these names should be more descriptive.  `Linux_x64_glibc` is ambiguous in this context.  Something like `Build_Linux_x64_glibx` would be more useful.

Reply:
Agreed, will take care this .